### PR TITLE
Restyle account page

### PR DESCRIPTION
**Pull request by AbdBarho**
Related #747 

image

### CODE REVIEW

File: website/src/components/Survey/SurveyCard.tsx [hunk 1]
```diff
@@ -1,22 +1,18 @@
...
-export const SurveyCard = (props: SurveyCardProps) => {
+export const SurveyCard = (props: PropsWithChildren<{ className?: string }>) => {
   const backgroundColor = useColorModeValue("white", "gray.700");
 
   const BoxClasses: BoxProps = {
     gap: "2",
     borderRadius: "xl",
     shadow: "base",
-    className: "p-4 sm:p-6",
+    className: "p-4 sm:p-6 " + (props.className ?? ""),
```
Comment (notmd) [inline website/src/components/Survey/SurveyCard.tsx:11]: You can use `clsx` here to simplify the code

File: website/src/pages/account/index.tsx [hunk 2]
```diff
@@ -1,9 +1,11 @@
-import { Button } from "@chakra-ui/react";
+import { Button, Divider, Flex, Grid, Icon, Text } from "@chakra-ui/react";
 import Head from "next/head";
 import Link from "next/link";
 import { useSession } from "next-auth/react";
 import React from "react";
 export { getDefaultStaticProps as getStaticProps } from "src/lib/default_static_props";
+import { MdOutlineEdit } from "react-icons/md";
```
Comment (notmd) [inline website/src/pages/account/index.tsx:7]: We are trying to use one type of icon. Can you switch to Feather icon here?

File: website/src/components/Survey/SurveyCard.tsx (hunk 1 above)
Comment (AbdBarho) [reply website/src/components/Survey/SurveyCard.tsx:11]: Of cousre!

File: website/src/pages/account/index.tsx (hunk 2 above)
Comment (AbdBarho) [reply website/src/pages/account/index.tsx:7]: I am not sure we talked about that.

Feather icon is not installed anyway, I will leave it to a different PR.

Comment (fozziethebeat) [reply website/src/pages/account/index.tsx:7]: I've been using Font Awesome (`react-icons/fa`) but they don't don't have an outline edit icon.  I think this is okay for now.

**Pull request merged**